cmake/FindDPCPP.cmake

```diff
@@ -41,6 +41,12 @@ set(CMAKE_CXX_FLAGS "${DPCPP_FP_FLAG} ${CMAKE_CXX_FLAGS}")
 # Disable range rounding feature to reduce # of SYCL kernels.
 set(CMAKE_CXX_FLAGS "-D__SYCL_DISABLE_PARALLEL_FOR_RANGE_ROUNDING__ ${CMAKE_CXX_FLAGS}")
 
+# Set flag to allow linking of large device code files. This option is currently
+# not available on Windows.
+if(NOT WIN32)
+    set(CMAKE_CXX_LINK_FLAGS "-fsycl-link-huge-device-code ${CMAKE_CXX_LINK_FLAGS}")
```

Comment:
FYI: this option is about to be replaced by `-flink-huge-device-code`. See https://github.com/intel/llvm/pull/9888.
We will need to update the name with the next DPC++ version update.